common/dconfig/configmgr.go

```diff
@@ -339,46 +369,75 @@ func (cfgMgr *CassandraConfigManager) mkKVTreeForSvc(service string, items []*m.
 			continue
 		}
 
-		versions[version] = struct{}{}
+		if sku != wildcardToken && host != wildcardToken {
+			key := strings.Join([]string{service,version, sku, host,cfgKey}, ".")
```

Comment:
cosmetic: the spaces between the items in the slice are inconsistent -- not sure why your editor/lint didn't fix it (mine did!)